tests/test_middleware.py

```diff
@@ -2,7 +2,7 @@
 from django.http import HttpResponse
 
 import mock
-from mohawk.exc import MacMismatch
+from mohawk.exc import MisComputedContentHash
```

Comment:
ok, this needs to change to `mohawk>=0.3.3` https://github.com/kumar303/hawkrest/blob/master/setup.py#L25